test/tests/fs/xhrFS/listing.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Unit tests for MountableFileSystem's mount/unmount features.
+ */
+var fs = require('fs'),
+    assert = require('wrapped-assert');
+
+module.exports = function() {
+  var oldRootFS = fs.getRootFS();
+
+  var rootForXFS = new BrowserFS.FileSystem.InMemory();
+  BrowserFS.initialize(rootForXFS);
+
+  var listing = {
+    "README.md":null,
+    "test": {
+      "fixtures": {
+        "static": {
+          "49chars.txt": null
+        }
+      }
+    },
+    "src":{
+      "README.md":null,
+      "backend":{"AsyncMirror.ts":null, "XmlHttpRequest.ts":null, "ZipFS.ts":null},
+      "main.ts":null
+    }
+  }
+
+  var newXFS = new BrowserFS.FileSystem.XmlHttpRequest(listing, "test/tests/fs/xhrFS");
+  rootForXFS.mount("/tRoot", newXFS);
```

Comment:
InMemoryFS doesn't have a `mount`. You should just make `newXFS` the root with `initialize` instead.